test/dlc_tests/ecs/mxnet/inference/test_ecs_mxnet_inference.py

```diff
@@ -19,7 +19,28 @@ def test_ecs_mxnet_inference_cpu(mxnet_inference, ecs_container_instance, region
         service_name, task_family, revision = ecs_utils.setup_ecs_inference_service(
             mxnet_inference, "mxnet", ecs_cluster_arn, model_name, worker_instance_id, region=region
         )
-        inference_result = request_mxnet_inference_squeezenet(public_ip_address)
+        inference_result = request_mxnet_inference(public_ip_address)
+        assert inference_result, f"Failed to perform inference at IP address: {public_ip_address}"
+
+    finally:
+        ecs_utils.tear_down_ecs_inference_service(ecs_cluster_arn, service_name, task_family, revision)
+
+
+@pytest.mark.model("squeezenet")
+@pytest.mark.parametrize("ecs_instance_type", ["c5.4xlarge"], indirect=True)
+@pytest.mark.parametrize("ecs_ami", [ECS_AML2_CPU_USWEST2], indirect=True)
```

Comment:
nit: it's fairly clear from the name, but can you also add `@pytest.mark.integration("elastic_inference")`